reference/v8.json

```diff
@@ -157,18 +157,18 @@
     },
     "maxzoom": {
       "type": "number",
-      "default": 14,
+      "default": 18,
       "doc": "Maximum zoom level at which to create vector tiles (higher means greater detail at high zoom levels)."
     },
     "buffer": {
       "type": "number",
-      "default": 64,
-      "doc": "Tile buffer size on each side (higher means fewer rendering artifacts near tile edges but slower performance)."
+      "default": 128,
+      "doc": "Tile buffer size on each side (relative to a 512px tile; higher means fewer rendering artifacts near tile edges but slower performance)."
```

Comment:
Yes. I just thought that simply mentioning that it's measured in pixels is not enough since it's not immediately apparent what a pixel is for a vector tile.